packages/api/test/e2e/rx-queries.spec.js

```diff
@@ -50,4 +50,22 @@ describe.skip('e2e queries', () => {
         done();
       });
   });
+
+  describe('linked map', () => {
+    it('queries state witha key', (done) => {
+      api.query.staking.test(1).subscribe((res) => {
+        console.log('!!', res)
+
+        done();
+      });
+    });
+
+    it.only('queries state head', (done) => {
+      api.query.staking.test.head().subscribe((res) => {
```

Comment:
I added 
`pub Test get(test) build(|_config| vec![(1, 2), (3, 4), (5, 6)]): linked_map u32 => u32;` to staking module for testing purpose.
I will change it to use `staking.validators` once I am confident it works